library/java/io/envoyproxy/envoymobile/engine/JniLibrary.java

```diff
@@ -17,7 +17,9 @@
   // WARNING: This should only be used for testing.
   public static void loadTestLibrary() {
     if (System.getProperty("envoy_jni_library_name") != null) {
-      envoyLibraryName = System.getProperty("envoy_jni_library_name");
+      envoyLibraryName = System.getProperty("os.name").startsWith("Linux")
```

Comment:
Not feeling good about having this in code but since this works and unblocks you, let's go with it. I can async try to understand why bazel is not cooperating like I expect it to.